src/status_im/ui/screens/network_settings/views.cljs

```diff
@@ -31,6 +35,9 @@
 (defn navigate-to-network [network]
   (re-frame/dispatch [:navigate-to :network-details {:networks/selected-network network}]))
 
+(defn navigate-to-add-network []
+  (re-frame/dispatch [:edit-network]))
```

Comment:
Curious about naming, why it's called `edit-network` why it's `add-network` essentially?  Also, actual editing is not possible at the moment.

Reply:
I chose to name the functionality "edit" as to future proof this and allow it to be re-used if editing is added in the future.